nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -479,13 +479,13 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.codehaus.groovy</groupId>
+            <groupId>org.apache.groovy</groupId>
             <artifactId>groovy-json</artifactId>
             <version>${nifi.groovy.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.codehaus.groovy</groupId>
+            <groupId>org.apache.groovy</groupId>
             <artifactId>groovy-servlet</artifactId>
             <version>${nifi.groovy.version}</version>
             <scope>test</scope>
```

Comment:
These dependency do not appear to be used in any tests in this module, and could be removed. There are two references to `groovy.time` classes in `TestEncryptContent` that should be removed with these modules.